src/js/components/Pagination/README.md

```diff
@@ -196,71 +197,349 @@ nav
-      xlarge: {
-        size: '26px',
-        height: '32px',
-        maxWidth: '624px',
-      },
-      xxlarge: {
-        size: '34px',
-        height: '40px',
-        maxWidth: '816px',
-      },
-    }
-```
-
-**pagination.extend**
-
-Any additional style for the Pagination. Expects `string | (props) => {}`.
+active-background
+```
+
+**pagination.button.color**
+
+The color of the text label. Expects `string | { dark: string, light: string }`.
+
+Defaults to
+
+```
+text-strong
+```
+
+**pagination.button.hover.background.color**
+
+Background color when the button when hovered. Expects `string | { dark: string, light: string }`.
+
+Defaults to
+
+```
+background-contrast
+```
+
+**pagination.button.hover.color**
+
+The color of the text label when hovered. Expects `string | { dark: string, light: string }`.
 
 Defaults to
 
 ```
 undefined
 ```
 
-**global.edgeSize**
+**pagination.container**
 
-The possible sizes for margin. Expects `object`.
+Any valid Box property affecting the Box 
+    wrapping the pagination controls. Expects `object`.
 
 Defaults to
 
 ```
-{
-    edgeSize: {
-      none: '0px',
-      hair: '1px',
-      xxsmall: '3px',
-      xsmall: '6px',
-      small: '12px',
-      medium: '24px',
-      large: '48px',
-      xlarge: '96px',
-      responsiveBreakpoint: 'small',
-    },
-  }
+undefined
+```
+
+**pagination.container.extend**
+
+Any additional style for the Box wrapping 
+    the pagination controls. Expects `string | (props) => {}`.
+
+Defaults to
+
+```
+undefined
+```
+
+**pagination.control.extend**
+
+Any additional style for each pagination control. Expects `string | (props) => {}`.
+
+Defaults to
+
+```
+undefined
+```
+
+**pagination.control.pad**
+
+Padding around each pagination control's label. Expects `string | object`.
+
+Defaults to
+
+```
+4px
+```
+
+**pagination.control.size.small.border.radius**
+
+Rounding of the corners for each control. Expects `string`.
+
+Defaults to
+
+```
+3px
+```
+
+**pagination.control.size.small.border.width**
+
+Border thickness for each control. Expects `string`.
+
+Defaults to
+
+```
+2px
+```
+
+**pagination.control.size.small.font.size**
+
+The font size of each control's label. Expects `string`.
+
+Defaults to
+
+```
+14px
+```
+
+**pagination.control.size.small.font.height**
+
+The line-height of each control's label. Expects `string`.
+
+Defaults to
+
+```
+20px
+```
+
+**pagination.control.size.small.height**
+
+The height for each pagination control. Expects `string`.
+
+Defaults to
+
+```
+30px
+```
+
+**pagination.control.size.small.width**
+
+The minimum width for each pagination control. 
+    Width will scale up fitting the control's label. Expects `string`.
+
+Defaults to
+
+```
+30px
+```
+
+**pagination.control.size.medium.border.radius**
+
+Rounding of the corners for each control. Expects `string`.
+
+Defaults to
+
+```
+4px
+```
+
+**pagination.control.size.medium.border.width**
+
+Border thickness for each control. Expects `string`.
```

Comment:
Should this specify "when pagination is size medium"?

Reply:
Do we need to repeat all the theme structures for each size -- is there any way to consolidate that?

Reply:
I wondered the same. I had looked at what we do on button, heading, etc. where sizes are used and the docs seems to be exhaustive. 

In this use case, I think we are trying to communicate preciseness and have an accurate inventory of the available properties (as opposed to documentation trying to convey a concept). This also allows for deep-links to be shared as well.

That said, I am open to any suggestions you have.

Reply:
I am comfortable leaving it as is then, especially if we do similarly in other areas.